Index: libgnomecanvas-2.30.3/libgnomecanvas/gnome-canvas-shape.c
===================================================================
--- libgnomecanvas-2.30.3.orig/libgnomecanvas/gnome-canvas-shape.c	2011-01-31 13:19:30.000000000 +0100
+++ libgnomecanvas-2.30.3/libgnomecanvas/gnome-canvas-shape.c	2011-07-30 17:04:59.907931567 +0200
@@ -1342,7 +1342,9 @@
 {
 	GnomeCanvasShapePrivGdk * gdk;
 
-	g_assert (!((GnomeCanvasItem *)shape)->canvas->aa);
+	if (((GnomeCanvasItem *)shape)->canvas != NULL) {
+		g_assert (!((GnomeCanvasItem *)shape)->canvas->aa);
+	}
 
 	gdk = shape->priv->gdk;
 
