Index: libgnomecanvas-2.30.3/demos/Makefile.am
===================================================================
--- libgnomecanvas-2.30.3.orig/demos/Makefile.am	2011-01-31 13:19:30.000000000 +0100
+++ libgnomecanvas-2.30.3/demos/Makefile.am	2013-12-27 14:36:33.084895344 +0100
@@ -12,6 +12,7 @@ INCLUDES = \
 	-DG_LOG_DOMAIN=\"GnomeCanvas\"
 
 LDADD =	\
+	$(LIBGNOMECANVAS_LIBS)				\
 	$(top_builddir)/libgnomecanvas/libgnomecanvas-2.la \
 	$(INTLLIBS)
 
